nixos/modules/services/desktops/pipewire/README.md

```diff
@@ -0,0 +1,6 @@
+# Updating
+
+1. Update the version & hash in pkgs/development/libraries/pipewire/default.nix
+2. run `nix build -I nixpkgs=/path/to/nixpkgs/checkout` nixpkgs.pipewire nixpkgs.pipewire.mediaSession
```

Comment:
```suggestion
2. run `nix build -f /path/to/nixpkgs/checkout pipewire pipewire.mediaSession`
```

Reply:
ty, that was the command that I'd been using but this one seems more ergonomic